bin/reth/src/node/events.rs

```diff
@@ -76,17 +79,19 @@ impl NodeState {
                 done,
             } => {
                 self.current_checkpoint = checkpoint;
+                self.eta.update(self.current_checkpoint);
 
                 if done {
                     self.current_stage = None;
+                    self.eta = Eta::default();
                 }
 
                 info!(
-                    target: "reth::cli",
                     pipeline_stages = %format!("{pipeline_position}/{pipeline_total}"),
                     stage = %stage_id,
-                    progress = checkpoint.block_number,
+                    block = checkpoint.block_number,
                     %checkpoint,
+                    eta = %self.eta,
```

Comment:
if I get it right, we will log `Stage finished executing` with `eta = "unknown"` on last iteration. Maybe let's split the `if done { ... } else { ... }` below into two separate `info!` loggings with no `eta` field in case of `done = true`?

Reply:
I'll just hoist the info up to before we reset our state.